openassessment/xblock/submission_mixin.py

```diff
@@ -637,3 +642,22 @@ def submission_path_and_context(self):
             path = 'openassessmentblock/response/oa_response_submitted.html'
 
         return path, context
+
+    def get_team_info(self):
+        teams_service = self.runtime.service(self, 'teams')
+        if not teams_service:
+            return None
```

Comment:
Do you think it's worth logging a warning or something here?

Reply:
actually, I wonder if it's also worth raising an exception here (and catching in the caller) to help distinguish between the case where the service isn't available/can't find the user, and where we have the service and the user, but the user isn't on a team yet - because we'll need to distinguish that latter case when we do the work to hide content from teamless users.